oshi-core-java24/src/main/java/oshi/hardware/HardwareAbstractionLayer.java

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright 2025 The OSHI Project Contributors
+ * SPDX-License-Identifier: MIT
+ */
+package oshi.hardware;
+
+public interface HardwareAbstractionLayer {
```

Comment:
We *could* copy over the `oshi-core` interface classes...

I don't see a reason for this empty interface though.